doc/index.rst

```diff
@@ -35,19 +32,41 @@ interface toolkits.
    </div>
    <span class="clear_screenshots"></span>
 
-Matplotlib tries to make easy things easy and hard things possible.  You
-can generate plots, histograms, power spectra, bar charts, errorcharts,
-scatterplots, etc., with just a few lines of code.  For examples, see the
-:doc:`sample plots <tutorials/introductory/sample_plots>` and :doc:`thumbnail
-gallery <gallery/index>`.
+Matplotlib makes easy things easy and hard things possible.
 
-For simple plotting the :mod:`.pyplot` module provides a MATLAB-like interface,
-particularly when combined with IPython.  For the power user, you have full
-control of line styles, font properties, axes properties, etc, via an object
-oriented interface or via a set of functions familiar to MATLAB users.
+.. container:: bullet-box-container
+
+   .. container:: bullet-box
+
+      Create
+
+      - Easy creation of standard plots
+      - Publication quality output to common file formats
+        (png, pdf, svg, ps, ...)
+      - Interactive figures (zoom, pan, ...)
+
+   .. container:: bullet-box
+
+      Customize
+
+      - Adapt and tune all aspects a graphic
+      - Embed in interactive environments (e.g. IPython_, Jupyter_)
+      - Create graphics that respond to user GUI events
+
+      .. _IPython: http://ipython.org
+      .. _Jupyter: http://jupyter.org
+
+   .. container:: bullet-box
+
+      Extend
+
+      - Use additional tailored functionality through
+        :doc:`third party packages <thirdpartypackages/index>`
+      - Embed in GUI applications (Qt, Tk, Gtk3, Wx)
+      - Serve from web servers
```

Comment:
Interactive, GUI stuff is already mentioned in create. And I have no attachment to that last bullet, was just trying to pull from stuff on the homepage and not repeat bullets. 
```
- Explore tailored functionality provided by :doc:`third party packages <thirdpartypackages/index>`
- Learn more about matplotlib through our many [external resources](https://matplotlib.org/resources/index.html)
```

Reply:
ok